client/allocrunner/alloc_runner.go

```diff
@@ -70,7 +72,7 @@ type AllocRunner struct {
 	updateCh chan *structs.Allocation
 
 	vaultClient  vaultclient.VaultClient
-	consulClient ConsulServiceAPI
+	consulClient consulApi.ConsulServiceAPI
```

Comment:
I think these vars would be named something like `discoClient discoAPI.Client` given my brainstorming above.